### ISSUE TITLE
Settings for one CoinJoin bundle

### ISSUE THREAD

**Issue by hynek-jina** [trezor/trezor-suite#6101]
Appears when user clicks on button to start CoinJoin from account.

![Screenshot 2022-09-20 at 10 05 56](https://user-images.githubusercontent.com/42465546/191202873-8d64a219-218c-4d77-b6d6-51e95ab89ac6.png)

- Recommended strategy is selected by default
- Amount is selected automatically - All funds that are not on required anonymity level and that are not assigned to another CoinJoin bundle

**Comment by komret** [comment]
Mostly resolved by #6306. TODOs:
- [x] Terms & Conditions and Privacy statement text
- [x] content of tooltips over the INFO icons (needs specification)
- [x] calculation of coin join related values (waiting for lower level implementation)
- [x] restructuring the page (will be done in a follow-up PR after #6306 and #6302 are merged)
- [x] correctly displaying amounts in the header (should be handled as part of #6322)

**Comment by komret** [comment]
**QA:** reopening the issue until #6354 is merged. Unless you already started testing this, let's wait for the other PR and possibly more functionality mentioned above.

**Comment by STew790** [comment]
**QA OK**
Settings for CJ bundle works as expected. Links for both terms are correct.

![image](https://user-images.githubusercontent.com/48477147/200325372-d176059d-fd55-4fab-b0de-334654ee2f19.png)
![image](https://user-images.githubusercontent.com/48477147/200325883-27edf2a0-d880-434b-aad3-3f940d703ad2.png)

**Info:**
 - Suite version: desktop 22.11.1 (8a8808d3c3fd752bde89507d50650bc4bac56647)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/22.11.1 Chrome/100.0.4896.160 Electron/18.3.5 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: model T 2.5.3 Universal (revision 2f03ace311584988d5aeab58fd1acf24ef54711a)
 - Transport: bridge 2.0.32
